aiven/table_aiven_account.go

```diff
@@ -0,0 +1,107 @@
+package aiven
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+func tableAivenAccount(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aiven_account",
+		Description: "Retrieve information about your accounts.",
+		List: &plugin.ListConfig{
+			Hydrate: listAccounts,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getAccount,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The account ID.",
+			},
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The account name.",
+			},
+			{
+				Name:        "billing_enabled",
+				Type:        proto.ColumnType_BOOL,
+				Description: "Check if the billing is enabled for the account.",
+			},
+			{
+				Name:        "owner_team_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The owner team ID of the account.",
+			},
+			{
+				Name:        "create_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The create time of the account.",
+			},
+			{
+				Name:        "update_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The update time of the account.",
+			},
+			{
+				Name:        "tenant_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The tenant ID of the account.",
+			},
+		},
+	}
+}
+
+func listAccounts(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	conn, err := getClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("listAccounts", "connection_error", err)
```

Comment:
```suggestion
		plugin.Logger(ctx).Error("aiven_account.listAccounts", "connection_error", err)
```

Reply:
@misraved `plugin.Logger(ctx).Error("listAccounts", "connection_error", err)` This error message is unique in the whole plugin; why do we need to add the table name here?